protobuf/coreapi/api.proto

```diff
@@ -402,6 +405,21 @@ message ServiceLogsRequest {
   repeated string dependencies = 2; // __Optional.__ List of dependencies to filter service logs. All by default.
 }
 
+// The request to fetch all informations of the Core
+message InfoRequest {}
+
+// Information concerning the Core from the `info` API.
+message InfoReply {
+  message CoreService {
+    string sid = 1;
+    string hash = 2;
+    string url = 3;
```

Comment:
Hash is useless because of sid.
The service key / type is missing.
The goal is for the cli to get the sid from a hardcoded type.
Eg: Give me SID of service type 'marketplace'.

Reply:
Ok for the "key" but the `hash` is absolutely needed, the `sid` can be deleted from that but the `hash` is the version we want to use, not the sid, sid is just an alias on the latest version deployed.
Sid is useless because of hash, not the other way around